### ISSUE TITLE
Add to master CSV file_downloadable

### ISSUE THREAD

**Issue by clapierre** [benetech/ImageShare#89]
For those resource files which are downloadable (ie. resource zip file, all images, all text files etc.)

@sinabahram what would not be considered downloadable?
I assume everything would be except those for streaming from another library right?
IE. all the DCMP video's would be not checked for downloadable.

Do I have that correct?

**Comment by clapierre** [comment]
@sinabahram if you are happy with only the DCMP streaming videos are the resource files that are not downloadable then the excel spreadsheet and UTF16 txt file has been updated with that information and the new column is file_downloadable with TRUE or FALSE being the only options.  Now if there is no file in the case of the DCMP resource file (since we don't have a zip file) I just left it blank there.

**Comment by sinabahram** [comment]
@clapierre, leaving the resource row's file URI empty makes sense for DCMP or for any resource where there is no "Download All" available. That may actually come in handy for other resources in the future as well. I will add on my list, though, to make an issue to not display "download all" where there is no file in resource object in JSON.

RE what is and isn't downloadable is simple. If you have it in S3, it's downloadable. Even if it's on another site, it can still be downloadable, and should be, except for when you cannot link directly to the file, exactly like DCMP. So DCMP is not downloadable, but if you were linking to an mp3 file on your personal website, for example, it would absolutely be downloadable.

**Comment by sinabahram** [comment]
@clapierre  I noticed you put "streaming" as the format. Is this accurate? For example, after one logs into DCMP, does one stream these videos or does one download them? If you can download them from DCMP, I don't think streaming should be the format. If you can't, and DCMP only allows streaming, then this is totally fine.

**Comment by clapierre** [comment]
Yes @sinabahram you can't download those videos they really are streaming, which is why I added this new File Format.
Thanks

**Comment by sinabahram** [comment]
Right on @clapierre . I'm done too. The scripts and such have been updated now, so I'm pinging @jkva to come along and close this issue once his portion is done. Job, support for zip file URI and downloadable boolean  have both been pushed. The URI field in resource now points to the zip file we discussed. Also, on the file level, the downloadable field has been added as a boolean. I'll make a separate issue, Job, for you to close for zip URI support.

**Comment by jkva** [comment]
@clapierre if the `downloadable` flag is set to `false`, will the `URI` property simply not exist for a resource, or will it contain a `NULL` value? I'm asking so I can configure the validation schema correctly.

**Comment by clapierre** [comment]
@jkva No there is a URI for a FALSE downloadable but you can't download that file, its a streaming video for example.
However for resources that don't contain an overarching zip file (ie. for videos that are streamed) then the downloadable flag is blank and the URI field is also blank.

**Comment by jkva** [comment]
@clapierre that's fine - in that case I can safely set both `downloadable` and `URI` to `required` 👍

**Comment by jkva** [comment]
Implemented via 7475e15a7c256997f86c3dc0cd61423be100f907
